app/src/main/java/org/dhis2/Bindings/ViewExtensions.kt

```diff
@@ -38,3 +41,20 @@ fun TextView.clearFocusOnDone() {
         }
     }
 }
+
+fun View.clipWithRoundedCorners(curvedRadio: Int = 16.dp) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+        outlineProvider = object : ViewOutlineProvider() {
```

Comment:
can we reouse this clip rounded in home screen?

Reply:
We can use it everywhere and dismiss the background. But i would create a separete issue for that

Reply:
Adding it in ANDROAPP-3372 as it will help to other bug